packages/js/components/src/experimental-select-tree-control/select-tree.tsx

```diff
@@ -179,6 +181,20 @@ export const SelectTree = function SelectTree( {
 					setInputValue( '' );
 					recalculateInputValue();
 				}
+			} else if (
+				( event.key === 'ArrowLeft' || event.key === 'Backspace' ) &&
+				( event.target as HTMLInputElement ).selectionStart === 0 &&
+				( event.target as HTMLInputElement ).selectionEnd === 0
+			) {
+				(
+					( event.target as HTMLElement )
+						.closest(
+							'.woocommerce-experimental-select-control__combo-box-wrapper'
+						)
+						?.querySelector(
+							'.woocommerce-experimental-select-control__selected-items > div:last-child button'
+						) as HTMLButtonElement
```

Comment:
This hardcodes a relationship between the experimental select control and the experimental select tree control. Is there such a hardcoded relationship? In other words, can the tree control be used by itself?